test-data/unit/check-expressions.test

```diff
@@ -1157,6 +1157,25 @@ b'%c' % (123)
 [case testUnicodeInterpolation_python2]
 u'%s' % (u'abc',)
 
+
+-- F-String
+-- --------
+
+
+[case testFStringParseOk]
+# flags: --fast-parser
+f'foobar'
+
+[case testFStringTypecheckExpression]
+# flags: --fast-parser
+# flags: --python-version 3.6
+a: str
+a = 'foo'
+b: str
+b = f'{a}bar'
+b == 'foobar'
```

Comment:
Oh, hm. It's supposed to pass in real life, but in the tests we're using a set of minimal stubs (see lib-stub/* and fixtures/*, and the `[builtins ...]` directove). I would just use a different test that doesn't use `==` -- it's not like the tests run this code, it's only type-checked.